src/visit_vtk/full/vtkVisItTubeFilter.C

```diff
@@ -104,7 +104,12 @@ int vtkVisItTubeFilter::RequestData(
     vtkIdType i;
     double range[2], maxSpeed=0;
     vtkCellArray *newStrips;
-    vtkIdType npts=0, *pts=NULL;
+    vtkIdType npts=0;
+#if LIB_VERSION_LE(VTK, 8,1,0)
+    vtkIdType *pts=NULL;
+#else
```

Comment:
`NULL` vs `nullptr` here. Does it matter? Can we make consistent?

Reply:
Should I changes all instances of 'NULL' in the file to 'nullptr' ?

Reply:
Though that might not be a bad idea, I am also not suggesting that either. But these two are really closely related and I thought it feels off to have they use different values for initialization.